### ISSUE TITLE
How to config normal mode mapping for all leaderf subcommands

### ISSUE THREAD

**Issue by linjiX** [Yggdroot/LeaderF#503]
From the doc, I found we can use `g:Lf_NormalMap` to set normal mode mapping.
But seems `g:Lf_NormalMap` can only set mappings for specific leaderf subcommand.
If I want to map `<C-j>` to `j` for all leaderf subcommands,
I have to repeat the same config in every subcommand in `g:Lf_NormalMap`.
```
let g:Lf_NormalMap = {
            \ "File":   [['<C-j>', 'j'], ['<C-k>', 'k']],
            \ "Buffer": [['<C-j>', 'j'], ['<C-k>', 'k']],
            \ "Mru":    [['<C-j>', 'j'], ['<C-k>', 'k']],
            \ "Tag":    [['<C-j>', 'j'], ['<C-k>', 'k']],
            \ ...
            \ ...
            \}
```

Is that an easy way to set normal mode mappings for all leaderf subcommands?

**Comment by tamago324** [comment]
I think so, too.

How about using `"*"` to do the base mapping?

```vim
let g:Lf_NormalMap = {
            \ "*":   [['<C-j>', 'j'], ['<C-k>', 'k']],
            \ ...
            \}
```

The topic changes, but it would be nice if you could define individual mappings in insert mode.

**Comment by linjiX** [comment]
I think using `"*"` is simple and very easy to understand.
Also it would be very good if `g:Lf_CommandMap` and `g:Lf_NormalMap` use the same config format.

**Comment by tamago324** [comment]
Implemented #566.

> `"_"` is a mapping that applies to all categories.
> Also, the mapping of each category overrides the mapping of `"_"`.
> 
> For example, to change the mapping for the File category only, do the following.
> 
> ```vim
> let g:Lf_NormalMap = {
> \   "_": [
> \      ['<C-j>', 'j'],
> \      ['<C-k>', 'k'],
> \   ],
> \   "File": [
> \      ['<C-j>', '<Nop>'],
> \      ['<C-k>', '<Nop>'],
> \   ],
> \}
> ```
